src/Keboola/DbExtractor/Extractor/Redshift.php

```diff
@@ -4,13 +4,21 @@
 
 namespace Keboola\DbExtractor\Extractor;
 
-use Keboola\Csv\CsvFile;
-use Keboola\Csv\Exception;
-use Keboola\DbExtractor\Exception\ApplicationException;
+use Keboola\Datatype\Definition\Redshift as RedshiftDatatype;
 use Keboola\DbExtractor\Exception\UserException;
 
 class Redshift extends Extractor
 {
+    public const INCREMENT_TYPE_NUMERIC = 'numeric';
+    public const INCREMENT_TYPE_TIMESTAMP = 'timestamp';
+    public const NUMERIC_BASE_TYPES = ['INTEGER', 'NUMERIC', 'FLOAT'];
+    public const TIMESTAMP_BASE_TYPES = [
+        'TIMESTAMP',
+        'TIMESTAMP WITHOUT TIME ZONE',
+        'TIMESTAMPTZ',
+        'TIMESTAMP WITH TIME ZONE',
+    ];
```

Comment:
what about datetime and date?

Reply:
datetime => timestamp in redshift
date -> added